scripts/benchmarks_ci.py

```diff
@@ -147,7 +147,8 @@ def __is_valid_datetime(dt: str) -> str:
     parser.add_argument('--upload-to-perflab-container',
         dest="upload_to_perflab_container",
         required=False,
-        help="Container to upload perf lab results to."
+        help="Container to upload perf lab results to.",
```

Comment:
Do we need default value here?

Reply:
good catch - need to update help. (though no one is ever actually going to use this but us.)